mon-pix/app/components/form-textfield.js

```diff
@@ -71,4 +73,11 @@ export default Component.extend({
     const inputValidationStatus = this.validationStatus;
     return MESSAGE_VALIDATION_STATUS_MAP[inputValidationStatus] || '';
   }),
+
+  actions: {
+    togglePasswordVisibility() {
+      this.toggleProperty('isPasswordVisible');
+      this.textfieldType === 'password' ? this.set('textfieldType', 'text') : this.set('textfieldType', 'password');
```

Comment:
désolée, je vais te faire rechanger, il y a encore plus court en fait :`this.set('textfieldType', this.textfieldType === 'password' ? 'text' : 'password' );` est plus clair ou pas selon toi ?

Reply:
Je pense que ta nouvelle proposition est bien, car on sait dès le départ quelle est l'intention.